tasks/e2e-behavior.sh

```diff
@@ -16,17 +16,17 @@ cd "$(dirname "$0")"
 # http://unix.stackexchange.com/a/84980
 temp_app_path=`mktemp -d 2>/dev/null || mktemp -d -t 'temp_app_path'`
 temp_module_path=`mktemp -d 2>/dev/null || mktemp -d -t 'temp_module_path'`
-custom_registry_url=http://localhost:4873
-original_npm_registry_url=`npm get registry`
-original_yarn_registry_url=`yarn config get registry`
+
+# Load Verdaccio-related functions
```

Comment:
Consolidated all Verdaccio related logic into one script that each of the e2e scripts now references (avoids a bunch of duplication).